ckanext/gla/templates/package/search.html

```diff
@@ -81,15 +79,3 @@
 {% endblock %}
 
 
```

Comment:
This is very nitpick-y, but in `page-search.html` there's a block called `secondary` that the `secondary_content` block goes into. So when we don't specify anything for the `secondary_content` block we still get the `<aside>` that's created in `{% block secondary %}` in `page-search.html`, but it's just empty:
![image](https://github.com/GreaterLondonAuthority/dfl-ckanext/assets/1217533/cdeb48e4-664e-4224-a861-6e06aa2d9d58)

Obviously no-one can see it and it doesn't really have any effect on the page layout, but we might as well get rid of it if we're not using it..?

I think this would do it, just to completely get rid of everything in the `secondary` block:
```suggestion
{% block secondary %}
{% endblock %}
```

(But equally I don't mind if we merge in what you've done, since it all looks fine 👍 )

Reply:
I did try that, but we actually have that block on other pages that use the same template, such as organisation search

Reply:
Edit... you were quite right! Somehow when I merged this it totally broke every link on the page. Gonna just push my fix to 1.x and you'll see what I mean